src/pages/Airplane/Airplane.tsx

```diff
@@ -42,7 +42,7 @@ const Airplane = () => {
 
 	const startLanding = (districtName: string) => {
 		const updatedMetadata = metadata.filter(t => {
-			!t.categories?.includes('District')
+			return !t.categories?.includes('District')
 		})
```

Comment:
We could use `getAvailableDistricts()` here instead. Maybe that should be a `useState` value set in a `useEffect`

Reply:
`getAvailableDistricts()` gets district options from `availableTraits`, whereas this removes filters out district from our current metadata.